nuxeo-drive-jsf/src/main/resources/OSGI-INF/nuxeodrive-actions.xml

```diff
@@ -104,6 +103,15 @@
       </rule>
     </filter>
 
+    <filter id="can_drive_edit_all">
```

Comment:
"All" is confusing, sound like we need to be able to edit all blobs.
=> `can_drive_edit_any`? `can_drive_edit_xpath`?